app/css/bootstrap.css

```diff
@@ -3235,20 +3235,18 @@ fieldset[disabled] .btn-success.focus {
 .btn-info:focus,
 .btn-info.focus {
   color: #fff;
-  background-color: #31b0d5;
-  border-color: #1b6d85;
+  background-color: #743dfb;
+
 }
 .btn-info:hover {
   color: #fff;
-  background-color: #31b0d5;
-  border-color: #269abc;
+  background-color: #743dfb;
```

Comment:
check if this change is making any difference - if not, get it back to how it was